crates/celo-revm/src/api/celo_block_env.rs

```diff
@@ -3,11 +3,17 @@ use op_revm::L1BlockInfo;
 use revm::primitives::Log;
 use std::vec::Vec;
 
+#[derive(Debug, Clone, Default)]
+pub struct Cip64Info {
+    pub actual_intrinsic_gas_used: u64,
+    /// Logs from system calls (debit/credit) that need to be merged into the final receipt
+    pub logs_pre: Vec<Log>,
+    pub logs_post: Vec<Log>,
+}
+
 #[derive(Debug, Clone, Default)]
 pub struct CeloBlockEnv {
     pub l1_block_info: L1BlockInfo,
     pub fee_currency_context: FeeCurrencyContext,
-    /// Logs from system calls (debit/credit) that need to be merged into the final receipt
-    pub cip64_actual_tx_system_call_logs_pre: Vec<Log>,
-    pub cip64_actual_tx_system_call_logs_post: Vec<Log>,
+    pub actual_cip64_tx_info: Cip64Info,
```

Comment:
I think we can find a better name. I without looking into the definition I would not know what the difference between actual and non-actual tx info would be.

Maybe something like
* cip64_effects
* cip64_side_effects
* cip64_results
* fee_currency_results
* fee_abstraction_output